src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -66,6 +66,7 @@ class Container implements ResettableContainerInterface
     protected $services = array();
     protected $methodMap = array();
     protected $privates = array();
+    protected $synthetics = array();
```

Comment:
they don't need to be listed in `getServiceIds` when they are not yet defined to me